langlib/lang.stream/src/main/ballerina/stream.bal

```diff
@@ -55,6 +61,11 @@ public isolated function filter(stream<Type,CompletionType> stm, @isolatedParam
 
 # Returns the next element in the stream wrapped in a record or () if the stream ends.
 #
+# ```ballerina
+# stream<string> greetingStream = ["Hello", "Bonjour", "Hola", "Ciao"].toStream();
+# greetingStream.next() ⇒ {"value":"Hello"}
```

Comment:
Shall we change this also to use a stream of numbers?

Reply:
yes, will do.